pyproject.toml

```
@@ -351,25 +351,33 @@ ignore_errors = true
 module = 'plumpy'
 
 [tool.pytest.ini_options]
-addopts = '--benchmark-skip --durations=50 --strict-config --strict-markers -ra --cov-report xml --cov-append '
+addopts = '--benchmark-skip --durations=50 --durations-min=1 --strict-config --strict-markers -ra --cov-report xml --cov-append '
 filterwarnings = [
+  'ignore:.*and will be removed in NumPy 2.0.*:DeprecationWarning:ase:',
+  'ignore:datetime.datetime.utcfromtimestamp:DeprecationWarning:pytz|dateutil|tqdm|aio_pika:',
+  'ignore:datetime.datetime.utcnow:DeprecationWarning:aio_pika|pytz|pgtest:',
   'ignore::DeprecationWarning:babel:',
   'ignore::DeprecationWarning:frozendict:',
   'ignore::DeprecationWarning:sqlalchemy:',
   'ignore::DeprecationWarning:yaml:',
   'ignore::DeprecationWarning:pymatgen:',
   'ignore::DeprecationWarning:jsonbackend:',
   'ignore::DeprecationWarning:pkg_resources:',
+  'ignore:ast.* is deprecated.*:DeprecationWarning:docstring_parser:',
+  'ignore::SyntaxWarning:CifFile:',
   'ignore::pytest.PytestCollectionWarning',
   'ignore:Creating AiiDA configuration folder.*:UserWarning',
   'ignore:Object of type .* not in session, .* operation along .* will not proceed:sqlalchemy.exc.SAWarning',
+  'ignore:Identity map already had an identity for .* inside of an event handler within the flush?:sqlalchemy.exc.SAWarning',
   'ignore:The `aiida.orm.nodes.data.upf` module is deprecated.*:aiida.common.warnings.AiidaDeprecationWarning',
   'ignore:The `Code` class is deprecated.*:aiida.common.warnings.AiidaDeprecationWarning',
   'default::ResourceWarning'
 ]
 markers = [
   'nightly: long running tests that should rarely be affected and so only run nightly',
   'requires_rmq: requires a connection (on port 5672) to RabbitMQ',
+  'requires_pgsql: requires a connection to PostgreSQL DB',
```

I would personally prefer `psql`. That seems to be the more common shorthand: https://trends.google.com/trends/explore?date=all&q=pgsql,psql&hl=en
```suggestion
  'requires_psql: requires a connection to PostgreSQL DB',
```